lighthouse-core/audits/dobetterweb/inspector-issues.js

```diff
@@ -136,11 +150,214 @@ class IssuesPanelEntries extends Audit {
+      const requestUrl = issue.request?.url;
+      if (requestUrl) {
+        requestUrls.add(requestUrl);
+      }
+    }
+    return {
+      issueType: str_(UIStrings.issueTypeCors),
+      subItems: {
+        type: 'subitems',
+        items: Array.from(requestUrls).map(url => ({url})),
+      },
+    };
+  }
+
+  /**
+   * @param {Array<LH.Crdp.Audits.DeprecationIssueDetails>} deprecationIssues
+   * @return {LH.Audit.Details.TableItem}
+   */
+  static getDeprecationRow(deprecationIssues) {
+    const urls = new Set();
+    for (const issue of deprecationIssues) {
+      const url = issue.sourceCodeLocation.url;
+      const lineNumber = issue.sourceCodeLocation.lineNumber;
+      urls.add(`${url}, line ${lineNumber}`);
+    }
+    return {
+      issueType: str_(UIStrings.issueTypeDeprecated),
+      subItems: {
+        type: 'subitems',
+        items: Array.from(urls).map(url => ({url})),
+      },
+    };
+  }
+
+  /**
+   * @param {Array<LH.Crdp.Audits.FederatedAuthRequestIssueDetails>} _federatedAuthReqIssues
+   * @param {string} url The url of the current page.
+   * @return {LH.Audit.Details.TableItem}
+   */
+  static getFederatedAuthRequestRow(_federatedAuthReqIssues, url) {
+    return {
+      issueType: str_(UIStrings.issueTypeFederatedAuthRequest),
+      subItems: {
+        type: 'subitems',
+        items: [{url: url}],
+      },
+    };
+  }
+
+  /**
+   * @param {Array<LH.Crdp.Audits.GenericIssueDetails>} _genericIssues
+   * @param {string} url The url of the current page.
+   * @return {LH.Audit.Details.TableItem}
+   */
+  static getGenericRow(_genericIssues, url) {
+    return {
+      issueType: str_(UIStrings.issueTypeGeneric),
+      subItems: {
+        type: 'subitems',
+        items: [{url: url}],
+      },
+    };
+  }
+
+  /**
+   * @param {Array<LH.Crdp.Audits.LowTextContrastIssueDetails>} lowTextContrastIssues
+   * @param {string} url The url of the current page.
+   * @return {LH.Audit.Details.TableItem}
+   */
+  static getLowTextContrastRow(lowTextContrastIssues, url) {
+    const urls = new Set();
+    for (const issue of lowTextContrastIssues) {
+      urls.add(`${url} - ${issue.violatingNodeSelector}`);
+    }
+    return {
+      issueType: str_(UIStrings.issueTypeLowTextContrast),
+      subItems: {
+        type: 'subitems',
+        items: Array.from(urls).map(url => ({url})),
+      },
+    };
+  }
+
+  /**
+   * @param {Array<LH.Crdp.Audits.NavigatorUserAgentIssueDetails>} navigatorUserAgentIssues
+   * @return {LH.Audit.Details.TableItem}
+   */
+  static getNavigatorUserAgentRow(navigatorUserAgentIssues) {
+    const urls = new Set();
+    for (const issue of navigatorUserAgentIssues) {
+      const url = issue.url;
+      const lineNumber = issue.location?.lineNumber;
+      urls.add(`${url}${lineNumber ? `, line ${lineNumber}` : ''}`);
+    }
+    return {
+      issueType: str_(UIStrings.issueTypeNavigatorUserAgent),
+      subItems: {
+        type: 'subitems',
```

Comment:
since this issue type has source line/col information, we can instead use `Audit.makeSourceLocation`. It should override the default column type here if you do this:

```
items: dedupedNavigatorUserAgentIssues.map(url => Audit.makeSourceLocation(url, line, column, bundle))
```

For an example of how to get the `bundle` value for this script (the script that url references), see `violation-audit`

Another option is to leave this for another PR / for us to handle (and just add items with the URL, no line/col info for now)